docs/sphinx/developers/jace/overview.txt

```diff
@@ -2,29 +2,34 @@ JACE C++ bindings for the Java API
 ==================================
 
 To make Bio-Formats accessible to software written in C++, we have
```

Comment:
To make the Bio-Formats Java classes accessible ... ? Or maybe something else mentioning Java BF?

Just wondering if there's some way we can make it clearer what this is about: after all, the native implementation is also to make BF accessible to software written in C++, albeit with rather fewer readers at present!